libkbfs/conflict_resolver.go

```diff
@@ -75,19 +75,19 @@ func NewConflictResolver(
 		},
 	}
 
-	cr.startProcessing()
+	cr.startProcessing(context.Background)
```

Comment:
I don't think you have to use a context-returning function; you can just use a context itself, since a context object is effectively immutable (all the `WithFoo` functions make copies).